src/Gtransfo.cc

```diff
@@ -263,22 +263,18 @@ class GtransfoInverse : public Gtransfo {
   GtransfoInverse(const GtransfoInverse&);
 
   //! Overload the "generic routine"
-  Gtransfo* RoughInverse(const Frame &Region) const
+  Gtransfo* RoughInverse(const Frame &) const
```

Comment:
As with the `*Star` classes, I don't want to touch gtransfo for now. These particular cleanups were to remove unused parameters to kill build warnings.